tests/random_utils.py

```diff
@@ -21,6 +21,9 @@ def random_int64():
 
 
 def random_bytes(n=100):
+    # if default size then return binary and printable data
+    if n == 100:
+        return bytes([i for i in range(n)])
```

Comment:
Interesting decision :) Maybe extract it into a separate function?

Reply:
when I tested locally I found that tests failed only when data contained `\` symbol with value `92` in ascii table. So I decided to include it for all binary values used by default (`random_bytes` function without parameters). Additionally, it includes printable [32; 127] values and not printable [0, 31]. If move it into a separate function then it requires updating all functions that use `random_bytes` function and decide which one needs it and which don't. But better to test such cases by default, imho.